modules/nw-ovn-kubernetes-migration.adoc

```diff
@@ -292,14 +273,31 @@ $ oc get pods --all-namespaces -o wide --sort-by='{.spec.nodeName}'
 +
 If pods on a node are in an error state, reboot that node.
 
+.. To confirm that all of the cluster Operators are not in an abnormal state, enter the following command:
++
+[source,terminal]
+----
+$ oc get co
+----
++
+The status of every cluster Operator must be the following: `AVAILABLE="True"`, `PROGRESSING="False"`, `DEGRADED="False"`. If a cluster Operator is not available or degraded, check the logs for the cluster Operator for more information.
+
 . Complete the following steps only if the migration succeeds and your cluster is in a good state:
 
-.. To remove the migration annotation from the Cluster Network Operator configuration object, enter the following command:
+.. To remove the migration configuration from the CNO configuration object, enter the following command:
++
+[source,terminal]
+----
+$ oc patch Network.operator.openshift.io cluster --type='merge' \
+  --patch '{ "spec": { "migration": null } }'
+----
+
+.. To remove custom configuration for the OpenShiftSDN network provider, enter the following command:
```

Comment:
Is there a reason that "OpenShiftSDN" is not spaced here as it is elsewhere? If not, I'd make it "OpenShift SDN". If it's referring to a literal value, should it be in backticks?